src/prelude/internals.mo

```diff
@@ -427,12 +430,13 @@ func @install_actor_helper(
         (#upgrade, (prim "cast" : (actor {}) -> Principal) actor2)
       }
     };
-  await @ic00.install_code({
+  await @ic00.install_code {
     mode;
     canister_id;
     wasm_module;
-    arg
-  });
+    arg;
+    sender_canister_version = ?(prim "canister_version" : () -> Nat64)()
```

Comment:
Do we need to recompute canister_version coz it could be different after the first await?

Reply:
Yes it will, see my other comment.